### PR TITLE
Salief/index pagination

### PR DESCRIPTION
**Pull request by salieflewis**
Ready for review.

### CODE REVIEW

File: apps/site/gql/queries/allAdds.graphql [hunk 1]
```diff
@@ -18,13 +23,16 @@ query allAdds {
...
-        }        
+        }
         adds(limit: 100, orderBy: "timestamp", orderDirection: "desc") {
           items {
             itemId
           }
         }
       }
     }
+    pageInfo {
```
Comment (0xTranqui) [inline apps/site/gql/queries/allAdds.graphql:34]: what do we get out of using a fragment here? feels like jsut adding complexity (more files to keep track of) without a gain of sorts

**Review by 0xTranqui — APPROVED** [review]
approving made one smol comment/q below, and leaving a few notes for follow up:
- think we want to figure out slightly more helpful pagination, that allows u to see what page index ur on (rather than just pure "prev/next")
- think we want to do something (related to first point) about how crazy the urls get when using the end cursor as a url param

File: apps/site/gql/queries/allAdds.graphql (hunk 1 above)
Comment (salieflewis) [reply apps/site/gql/queries/allAdds.graphql:34]: We're going to want to use this fragment across multiple queries. So the gain is we can do that from now on.

**Comment by salieflewis** [comment]
> approving made one smol comment/q below, and leaving a few notes for follow up:
> 
> * think we want to figure out slightly more helpful pagination, that allows u to see what page index ur on (rather than just pure "prev/next")
> * think we want to do something (related to first point) about how crazy the urls get when using the end cursor as a url param

Honestly what makes the most sense, thinking about this even more, is just infinite scroll.

**Pull request merged**